owncloudApp/src/main/res/xml/locales_config.xml

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="utf-8"?>
+<locale-config xmlns:android="http://schemas.android.com/apk/res/android">
+    <locale android:name="af"/> <!-- Afrikaans (South Africa) -->
+    <locale android:name="am"/> <!-- Amharic (Ethiopia) -->
+    <locale android:name="ar"/> <!-- Arabic (Arab World) -->
+    <locale android:name="arn"/> <!-- Mapudungun (Chile) -->
+    <locale android:name="as"/> <!-- Assamese (India) -->
```

Comment:
We should place here first at least the 3 ultimate fallback languages, these are: `en`, `de` and `es`